lisa/tests/test_variables.py

```diff
@@ -0,0 +1,285 @@
+import os
+from pathlib import Path
+from typing import Any, Dict, cast
+from unittest.case import TestCase
+
+from lisa import secret, variable
+from lisa.util import LisaException, constants
+from lisa.util.logger import get_logger
+
+
+class VariableTestCase(TestCase):
+    def setUp(self) -> None:
+        secret.reset()
+
+    def test_in_env(self) -> None:
+        os.environ["LISA_normalValue"] = "value_from_env"
+        os.environ["S_LISA_normalEntry"] = "s_value_from_env"
```

Comment:
How much configuration are we really expecting to set via environment variables? LISAv2 takes almost no input in this manner, instead preferring explicitly set variables on the command line.

Reply:
I feel env variables are more secure than command line. The command line are copy/paste, and easy to disclose. Github Action and ADO supports to pass secret values through env variables. That keeps env variables not be touched by command line users.